Documentation/Blazorise.Docs/Services/ThemeService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+namespace Blazorise.Docs.Services;
+
+public class ThemeService
```

Comment:
Make sure that the accessors properly encapsulate logic, just for good practices sake, if you care about it, otherwise it's fine.
i.e **SystemIsDarkMode** can be publicly set directly.